CMakeModules/FindNCCL.cmake

```diff
@@ -0,0 +1,24 @@
+# Find the NCCL libraries
+#
+# The following variables are optionally searched for defaults
+#  NCCL_ROOT_DIR:    Base directory where all NCCL components are found
+#
+# The following are set after configuration is done:
+#  NCCL_FOUND
+#  NCCL_INCLUDE_DIR
+#  NCCL_LIBRARY
+
+find_path(NCCL_INCLUDE_DIR NAMES nccl.h
+    PATHS ${NCCL_ROOT_DIR}/include
```

Comment:
Are there any "standard" path where it is most likely to be found (like /usr/local/cuda, or something like that).